.gitignore

```diff
@@ -20,6 +20,9 @@ playground/*
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
+.DS_Store
+app.py
```

Comment:
Why have you added app.py?

Reply:
We have created the script app.py for testing perposes then excluded it from git. So it might be better to ignore the changes in .gitignore when merging into the master.